libfreerdp/core/gateway/tsg.c

```diff
@@ -256,7 +253,6 @@ BOOL TsProxyCreateTunnelReadResponse(rdpTsg* tsg, RPC_PDU* pdu)
 		{
 			offset += 4; /* MsgID */
 			offset += 4; /* MsgType */
-			IsMessagePresent = *((UINT32*) &buffer[offset]);
```

Comment:
I agree that the variable is not used, but removing it makes you forget there was something there in the first place. These kind of variables are just there to correctly contain data parsed from the packet, even if currently unused.